client/models/artwork.coffee

```diff
@@ -17,3 +17,39 @@ module.exports = class Artwork extends Backbone.Model
 
   defaultImage: ->
     new AdditionalImages(@get('images')).default()
+
+  denormalized: ->
+    type: 'artwork'
+    id: @get('_id')
+    slug: @get('id')
+    date: @get('date')
+    title: @get('title')
+    image: @defaultImage().bestImageUrl(['larger','large', 'medium', 'small'])
+    partner:
+      name: @getPartnerName()
+      slug: @getPartnerLink()
+    artist:
+      name: @getArtistName()
+      slug: @get('artist')?.id
+
+  getArtistName: ->
```

Comment:
:+1:

Although these might be something we want to [share across Backbone apps](https://github.com/artsy/artsy-backbone-mixins/tree/master/lib), or leverage Metaphysics for. Totally cool with this implementation for now though.